internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,444 @@
+}
+
+func newImportCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import <spec.yaml>",
+		Short: "Adds topics, schemas and tags from the yaml file.",
+		Args:  cobra.ExactArgs(1),
+		Example: examples.BuildExampleString(
+			examples.Example{Text: "Import an asyncapi specification file to populate an existing cluster",
+				Code: "confluent asyncapi import spec.yaml",
+			},
+		),
+	}
+	c := &command{pcmd.NewAuthenticatedStateFlagCommand(cmd, prerunner), nil}
+	cmd.RunE = c.parse
+	cmd.Flags().Bool("overwrite", false, "Overwrite existing topics and schemas.")
+	cmd.Flags().String("kafka-api-key", "", "Kafka cluster API key.")
+	cmd.Flags().String("schema-registry-api-key", "", "API key for Schema Registry.")
+	cmd.Flags().String("schema-registry-api-secret", "", "API secret for Schema Registry.")
+
+	return cmd
+}
+
+func getFlagsImport(cmd *cobra.Command) (*flagsImport, error) {
+	overwrite, err := cmd.Flags().GetBool("overwrite")
+	if err != nil {
+		return nil, err
+	}
+	kafkaApiKey, err := cmd.Flags().GetString("kafka-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiKey, err := cmd.Flags().GetString("schema-registry-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiSecret, err := cmd.Flags().GetString("schema-registry-api-secret")
+	if err != nil {
+		return nil, err
+	}
+
+	flags := &flagsImport{
+		overwrite:               overwrite,
+		kafkaApiKey:             kafkaApiKey,
+		schemaRegistryApiKey:    schemaRegistryApiKey,
+		schemaRegistryApiSecret: schemaRegistryApiSecret,
+	}
+	return flags, nil
+}
+
+func (c *command) parse(cmd *cobra.Command, args []string) error {
+	spec, err := fileToStruct(args[0])
+	if err != nil {
+		return err
+	}
+	// Get flags
+	flagsImp, err := getFlagsImport(cmd)
+	if err != nil {
+		return err
+	}
+	// Getting Kafka Cluster & SR
+	flags := createExportFlagsFromImportFlags(flagsImp)
+	details, err := c.getAccountDetails(&flags)
+	if err != nil {
+		return err
+	}
+	for topicName, topicDetails := range spec.Channels {
+		err = c.addChannelToCluster(details, spec, topicName, topicDetails.TopicBinding.Kafka, false, flagsImp.overwrite)
+		if err != nil {
+			if err.Error() == parseErrorMessage {
+				log.CliLogger.Info(err)
+			} else {
+				log.CliLogger.Warn(err)
+			}
+		}
+	}
+	output.Printf("AsyncAPI specification imported from \"%s\"\n", args[0])
+	return nil
+}
+
+func fileToStruct(fileName string) (*Spec, error) {
+	asyncSpec, err := os.ReadFile(fileName)
+	if err != nil {
+		return nil, err
+	}
+	spec := new(Spec)
+	err = yaml.Unmarshal(asyncSpec, &spec)
+	if err != nil {
+		return nil, fmt.Errorf("unable to unmarshal yaml file: %v", err)
+	}
+	return spec, nil
+}
+
+func (c *command) addChannelToCluster(details *accountDetails, spec *Spec, topicName string, kafkaBinding KafkaBinding, topicNotCreated, overwrite bool) error {
+	output.Printf("Importing topic: %s\n", topicName)
```

Comment:
This message is better suited to be logged. Our CLI only prints the end result, not intermediate steps.

Reply:
There can be hundreds of topics in the input file. This is being printed to tell the user that the code is in progress and not stuck. Similar to how we are printing `Adding operation` in the export command.

Reply:
How long does this command take to run if there are hundreds of topics?

Reply:
Also, I think I understand what this command is doing a little better now. Let's try to mirror some of the other commands we have, and print messages like this one in the following format:
```
Created topic "my-topic".
```

This would look something like `output.Printf(errors.CreatedResourceMsg, resource.Topic, topicName)`

Reply:
It takes around 3 seconds per topic.